aten/src/ATen/native/Math.h

```diff
@@ -2177,4 +2177,48 @@ static inline C10_HOST_DEVICE T calc_log_ndtr(T x) {
   }
 }
 
+template<typename T>
+static inline T chebyshev_polynomial_t_forward(T x, std::int64_t n) {
```

Comment:
ROCM error is likely because this function is not marked C10_HOST_DEVICE

Reply:
Thank you @ngimel! That resolved the issue. :)